### PR TITLE
phpunit (instead of simple-phpunit)

### PR DESCRIPTION
**Pull request by jaspernbrouwer**
**Please review & merge #223 first.**

This PR installs PHPunit as dev-dependency, removes additions made by symfony/phpunit-bridge (but keep it for the deprecation notices), and makes the test-scripts use phpunit itself (instead of simple-phpunit).

This solves:
- phpunit being installed separately in each Travis build.
- IDE's missing classes some classes or seeing some double.

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -11,6 +11,10 @@
         <env name="SHELL_VERBOSITY" value="-1" />
     </php>
 
+    <listeners>
```
Comment (ramondelafuente) [inline phpunit.xml]: I think PHPunit parses the .dist file if you don't  have a local version, right? (so there is no need to rename and add to the repo, you can just change this in the .dist and leave the option to override if some wants to test things differently on a local environment)

**Comment by ramondelafuente** [comment]
(I changed the base of your PR)

File: phpunit.xml (hunk 1 above)
Comment (jaspernbrouwer) [reply phpunit.xml]: :+1: I'll rename it (back) to `phpunit.xml.dist`!

**Comment by jaspernbrouwer** [comment]
I messed up this PR rebasing. I'll close this one and submit a new one. Sorry!

**Pull request closed without merging**